indico/modules/events/surveys/util.py

```diff
@@ -59,3 +67,46 @@ def was_survey_submitted(survey):
     if submission_id is None:
         return False
     return bool(SurveySubmission.find(id=submission_id).count())
+
+
+def generate_csv_from_survey(survey, submitter_ids):
+    """Generates a CSV file from a given survey.
+
+    :param submitter_ids: The list of `User` ids. Generated file includes only submissions made by those users
+    """
+    fieldnames = {'submitter', 'submission_date'}
+    fieldnames |= {'{}_{}'.format(question.title, question.id) for question in survey.questions
+                   if not isinstance(question.field, StaticTextField)}
+
+    with NamedTemporaryFile(suffix='indico.tmp', dir=Config.getInstance().getTempDir(), delete=False) as temp_file:
+        submissions = _filter_submissions(survey, submitter_ids)
+        writer = csv.DictWriter(temp_file, fieldnames=fieldnames)
+        writer.writeheader()
+        for submission in submissions:
+            submission_dict = {
+                'submitter': (submission.user.full_name.encode('utf-8')
+                              if submission.user else 'Anonymous'),
```

Comment:
Wouldn't an empty column make more sense? I think when processing this data in excel etc. having no value if there is no value is more useful than having something that almost looks like a value